xep-0060.xml

```diff
@@ -2940,11 +2946,11 @@ And by opposing end them?
   </pubsub>
 </iq>
 ]]></example>
-      <p>The &lt;publish-options/&gt; element SHOULD contain a data form (see <cite>XEP-0004</cite>), whose FORM_TYPE SHOULD be "http://jabber.org/protocol/pubsub#publish-options" (see <cite>XEP-0068</cite>).</p>
-      <p>How the fields are to be handled is up to the the pubsub service, which in the language of XEP-0004 functions as a form-processing entity.</p>
-      <p>For example, the service may treat the field as a precondition, in which case the service should proceed as follows:</p>
+      <p>The &lt;publish-options/&gt; element MUST contain a data form (see <cite>XEP-0004</cite>), whose FORM_TYPE MUST be "http://jabber.org/protocol/pubsub#publish-options" (see <cite>XEP-0068</cite>).</p>
+      <p>Fields and their behaviour MUST be registered. Each field MUST specify whether it defines METADATA to be attached to the item, a per-item OVERRIDE of the node configuration, or a PRECONDITION to be checked against the node configuration. A pubsub service advertising support for publishing options MUST reject publications with unknown fields.</p>
```

Comment:
I'd probably write where the fields and their behavior must be registered, like it's done in other places in the XEP. So instead of

> Fields and their behaviour MUST be registered.

i'd write

> Fields and their behaviour MUST be registered with the XMPP Registrar.